kube/src/lib.rs

```diff
@@ -169,24 +168,21 @@ pub use kube_derive::CustomResource;
 #[cfg_attr(docsrs, doc(cfg(feature = "derive")))]
 pub use kube_derive::Resource;
 
-/// Re-exports from `kube-runtime`
 #[cfg(feature = "runtime")]
 #[cfg_attr(docsrs, doc(cfg(feature = "runtime")))]
 #[doc(inline)]
 pub use kube_runtime as runtime;
 
 pub use crate::core::{CustomResourceExt, Resource, ResourceExt};
-/// Re-exports from `kube_core`
-#[doc(inline)]
-pub use kube_core as core;
```

Comment:
I think this will be overridden by `just fmt`

Reply:
it was actually done by `just fmt` because - maybe it saw it could fit in the [single line limit](https://github.com/kube-rs/kube/blob/main/rustfmt.toml#L8) without the doc :thinking: